index.js

```diff
@@ -1,102 +1,87 @@
 import './sass/index.scss'
-
-import { AniX } from 'anix'
-
-const $ = require('jquery')
-window.$ = window.jQuery = $
-require("jquery-ui-bundle")
+import '@fortawesome/fontawesome-free/css/all.css'
+import $ from 'jquery'
 
 // Hiring message
 const hiringMessage = `Hey, you.
 You’re finally awake.
 You were trying to see the code, right?
 Walked right into that hiring message, same as us.
-If you’d like to work on this website and other cool projects with hackathons, send an email over to engineering@coderit.org!`
+If you’d like to work on hackathon-related projects, check out https://brickhack.io/club!`
 
-console.log(hiringMessage)
+console.log(hiringMessage);
 
 // Comment generated via js instead of directly in HTML so the hiring message text is only in one place
-const comment = document.createComment("\n"+hiringMessage.toString()+"\n")
-document.insertBefore(comment, document.firstChild)
-
-// Secret messages
-const revertDuration = 300
-var messageIndex = 0
-const messages = [
-    "Like, really, nothing to see here.",
-    "Hey. Stop looking.",
-    "Hey. Stop looking.",
-    "...",
-    "I'm warning you...",
-    "There's really nothing.",
-    "There's really nothing.",
-    "There's really nothing.",
-    "...",
-    "Wow, you just don't want to believe me.",
-    "What have I ever done to you?!",
-    "...",
-    "Oh, right...",
-    "...the bricks...",
-    "...",
-    "Yeah, that was... my bad.",
-    "Life's hard as a rockstar, you know?",
-    "I'm really a brickstar, actually.",
-    "... leader of ThunderClan? I don't get the reference.",
-    "Have you heard my new song?",
-    "It goes like,",
-    "I'm bringin' Ricky Back! (yeah)",
-    "(barrrw) Them other boys don't know--",
-    "I'M BRINGIN' RICKY BACK (yeah)",
-    "It's a bop, if I do say so myself.",
-    "Oh, by the way Dave,",
-    "We're contacting you about your car's extended warrany...",
-    "Haha, just kidding.",
-    "I have some brick business to get back to.",
-    "You know how it is.",
-    "I do have a tip for you though.",
-    "When you sign up for BrickHack...",
-    "...make sure to ask if you need your own bricks.",
-    "Heh. Gets them every year.",
-    "Well, see ya!",
-    "(the wall is quiet.)",
-    "(it fills you with determination.)",
-    ""
-]
+const comment = document.createComment("\n"+hiringMessage.toString()+"\n");
```

Comment:
nit: space between operators and operands
```suggestion
const comment = document.createComment("\n" + hiringMessage.toString() + "\n");
```

Reply:
Also, isn't `hiringMessage` already a string? The call to `String.toString()` feels redundant here.

Reply:
Fixed spaces in upcoming fixes commit. 
hiringMessage is technically a template literal and not a string (see usage of backtick instead of any quotes) which is why I'm guessing peter initially threw in a toString, but I checked and there is no visible change when removing that function call

Reply:
A template literal *is* a string, which is why there's no change from removing `toString()`.